odoo/tools/mail.py

```diff
@@ -639,6 +639,13 @@ def email_escape_char(email_address):
 def decode_message_header(message, header, separator=' '):
     return separator.join(h for h in message.get_all(header, []) if h)
 
+def contact_formataddr(pair, charset='utf-8'):
```

Comment:
why not doing it in `formataddr` directly ? (so the foramting is consistent everywhere)